app/src/main/java/com/example/marvelapp/util/TransformExtension.kt

```diff
@@ -4,14 +4,18 @@ import com.example.marvelapp.database.entity.CharacterEntity
 import com.example.marvelapp.entity.Character
 import com.example.marvelapp.service.response.DataResponse
 
+private const val DOT = "."
+
 fun DataResponse.transformToList(): List<Character> {
     val characterList = mutableListOf<Character>()
-    data.characters.forEach() { characterList.add(Character(it.id, it.name, it.description)) }
+    data.characters.forEach() {
+        characterList.add(Character(it.id, it.name, it.description, (it.img.path+DOT+it.img.ext)))
```

Comment:
use a more kotlin style way to concatenate the strings, please

"${it.img.path}$DOT${it.img.ext}"

Reply:
done